plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2436,6 +2437,11 @@ read_only::get_account_results read_only::get_account( const get_account_params&
    result.cpu_limit = rm.get_account_cpu_limit_ex( result.account_name, greylist_limit).first;
    result.ram_usage = rm.get_account_ram_usage( result.account_name );
 
+   producer_plugin* producer_plug = app().find_plugin<producer_plugin>();
```

Comment:
The find_plugin is actually rather expensive. This should be setup in plugin init or startup and asserted to be there. Assuming you don't change this to use plugin_interface methods.

Reply:
Thanks. As this is in read_only class, I will initialize it in read_only constructor. I did this too rush last night.